experimental/dds/tree/src/EditLog.ts

```diff
@@ -527,7 +528,8 @@ export class EditLog<TChange = unknown> extends TypedEventEmitter<IEditLogEvents
 	}
 
 	private evictEdits(): void {
-		const minSequenceIndex = this.sequenceNumberToIndexMap.get(this._minSequenceNumber) ?? 0;
+		const closestPair = this.sequenceNumberToIndex.getPairOrNextLower(this._minSequenceNumber);
+		const minSequenceIndex = closestPair !== undefined ? closestPair[1] : 0;
```

Comment:
```suggestion
		const minSequenceIndex = closestPair?.[1] ?? fail("No index associated with sequence number");
```
You know that the entry will exist because you started the map with [0, 0], `_minSequenceNumber` starts at 0, and `_minSequenceNumber` only gets bigger - we know because there is an assert on line 476 that ensures it is mOnOToNiCaLly increasing :). And we only delete ranges less than the `_minSequenceNumber`. So, if we hit this case, it's a btree bug, and defaulting to 0 at worst will obfuscate the problem.

Reply:
Oooo I was wondering how to make that syntax work. Thanks!